src/q7/instrusensors-ELEC2811/summary/L3/noise&calibration.tex

```diff
@@ -0,0 +1,169 @@
+\section{Noise and calibration}
```

Comment:
This is a dangerous file name. I would prefer if is was named "noise_and_calibration.tex"